README.rst

```diff
@@ -88,55 +87,60 @@ Records can be retrieved from and saved to collections.
 
-    # Get all records
-    record = collection.get_records()
-    record = collection.get_record(id='89881454-e4e9-4ef0-99a9-404d95900352')
-    collection.save_record(record)
-    collection.save_records([record1, record2])
-    collection.delete_record(id='89881454-e4e9-4ef0-99a9-404d95900352')
-    collection.delete_records([record1, record2])
+    # Retrieve a specific record and update it.
+    record = client.get_record('89881454-e4e9-4ef0-99a9-404d95900352',
+                               collection='todos', bucket='default')
+    client.update_record(record, collection='todos', bucket='default')
 
-    # Alternative use
-    record.save()
+    # Update multiple records at once.
+    client.update_records(records, collection='todos')
 
+    # It is also possible to delete records.
+    client.delete_record(id='89881454-e4e9-4ef0-99a9-404d95900352',
+                         collection='todos')
 
 Permissions
 ===========
 
- By default, authenticated users will get read and write access to the
- manipulated objects. It is possible to change this behavior by passing a dict
- to the `permissions` parameter.
+ By default, authors will get read and write access to the manipulated objects.
+ It is possible to change this behavior by passing a dict to the `permissions`
+ parameter.
 
  .. code-block:: python
 
-    record = collection.create_record(
-        data={},
-        permissions={'read': ['group:groupid']})
+    client.create_record(data={}, permissions={'read': ['group:groupid']},
+                         collection='todos')
 
 .. note::
 
     Every creation or modification operation on a distant object can be given
     a `permissions` parameter.
 
-The `Bucket`, `Collection`, `Group` and `Record` classes have a special
-`permissions` object that can be mutated in order to update the permissions
-model attached to the object.
-
-.. code-block:: python
-
-    bucket = Bucket('default', auth=('alexis', 'p4ssw0rd'))
+Buckets, Collections and Groups and records have permissions which can be
+edited.
 
-    friends = ['natim', 'niko', 'mat', 'tarek']
-    bucket.permissions.write += friends
-    bucket.permissions.create_collection += friends
+  # Different proposals below.
+  # 1. Change the API to return the permissions when asked, in a separate
+  # object.
+  record, permissions = client.get_record(1234, collection='todos',
+                                          include_permissions=True)
+  client.update_record(record, permissions=permissions, collection='todos')
 
-    # You *need* to call save in order to have these changes reflected in the
-    # remote.
-    bucket.save()
+  # 2. Allow the mutation of the permissions object, attached to a record.
+  record = client.get_record(1234, collection='todos')
+  record.permissions.write += ['leplatrem', ]
+  client.update_record(record)
 
+  # In any case, for the creation it will be possible to pass the permissions.
+  client.create_record(record, permissions={})
```

Comment:
It is important to define what a record is here.

It looks obvious for me to keep the resource as:
```
{
    "data": {
        "id": "bde49941-9879-cff1-3fab-c4a8e1aa7d31", 
        "last_modified": 1443711864134
    }, 
    "permissions": {
        "write": [
            "basicauth:f68bbb4c2d6b9f2d73609054ba6f18876c56cc1a039cd23e6ffdac1fce637dba"
        ]
    }
```

But sometimes you are talking about record as if it was record.data:
```
    {
        "id": "bde49941-9879-cff1-3fab-c4a8e1aa7d31", 
        "last_modified": 1443711864134
    }
```